TileStache/Goodies/VecTiles/transform.py

```diff
@@ -291,44 +291,45 @@ def road_classifier(shape, properties, fid, zoom):
 def road_sort_key(shape, properties, fid, zoom):
     # Note! parse_layer_as_float must be run before this filter.
 
-    # Calculated sort value is in the range 0 to 39
-    sort_val = 0
+    floor = 300
+    ceiling = 385
```

Comment:
Based on ±50 for bridges and tunnels, I suspect 447 would be a better ceiling.

300 base + 5 + [50-92 for basic roads + gondolas plus spacers] + 50 + 5 = 

* _some natural buffer for things like buildings (381-50=331, and we don't allow aerialway to be tunnels)_
* road layer default (paths, steps, tracks, etc)	355
* other road service (alley, driveway)	356
* other railway service	357
* unclassified, service, minor	358
* railway yard	359
* residential, unclassified, road, living_street, Unknown	360
* railway spur, siding	361
* _reserved for water lines & other goodies	362-374_
*_link	375
* tertiary, Track', taxiway	376
* secondary, runway	377
* primary, Road'	378
* trunk, Secondary Highway'	379
* rail, tram, light_rail, narrow_guage, monorail	380
* motorway, 'Major Highway', 'Beltway, 'Bypass'	381
* _ reserved for platforms, water lines, etc	372-389_
* other aerialway	390
* chair_lift	391
* gondola, cable_car	392